client/my-sites/site-settings/form-writing.jsx

```diff
@@ -123,6 +124,23 @@ class SiteSettingsFormWriting extends Component {
 						<PressThis />
 					</div>
 				) }
+				{
+					this.props.isJetpackSite && this.props.jetpackSettingsUISupported && (
+						this.renderSectionHeader( translate( 'Media' ) ) )
+				}
+				{
+					this.props.isJetpackSite && this.props.jetpackSettingsUISupported && (
+						<MediaSettings
+							siteId={ this.props.siteId }
+							onSubmitForm={ this.props.handleSubmitForm }
```

Comment:
haha, this is for copypasting the props passed to Composing. Looks like it's not used there either